module/rdpRandR.c

```diff
@@ -532,29 +577,40 @@ rdpRRSetRdpOutputs(rdpPtr dev)
            pRRScrPriv->numCrtcs, pRRScrPriv->numOutputs, dev->monitorCount));
     if (dev->monitorCount <= 0)
     {
-        left = 0;
-        top = 0;
-        width = dev->width;
-        height = dev->height;
+        struct monitor_info monitor;
+        memset(&monitor, 0, sizeof(monitor));
+        monitor.top = 0;
+        monitor.left = 0;
+        monitor.right = dev->width - 1;
+        monitor.bottom = dev->height - 1;
+        monitor.physical_width
+            = clientCon->client_info.display_sizes.session_width;
+        monitor.physical_height
+            = clientCon->client_info.display_sizes.session_height;
```

Comment:
I can see where your comment about putting the physical size into neutrinolabs/xrdp#2310 is coming from now.

As I said in response to your comment, that only makes sense in a single monitor scenario. Let me know what your thoughts are in this area.